x/programs/cmd/simulator/src/lib.rs

```diff
@@ -190,7 +190,7 @@ pub struct PlanResult {
     /// The timestamp of the function call response.
     pub timestamp: u64,
     /// The result of the function call.
-    pub response: Option<Vec<i64>>,
+    pub response: Option<String>,
```

Comment:
Seems to be how a `[]byte` gets serialized... we need to get away from json...

Reply:
🤔 `serde_as` does so much more than we need it to. We can just use `serde(with = "...")` and manually convert back and forth to a `String`. We likely will never need more than a borrow here though, so we can just use `String::as_bytes` when we do a comparison.